### PR TITLE
Chore: Update `new-rule`.

### LINKED ISSUE #1013: Update new-rule docs generation
Few things to update:

* [x] Sort properties in package.json
* [x] Update documentation
* [x] For packages multirules, generate one doc file per rule, and update the README.md

### PR DESCRIPTION
**Pull request by sarvaje**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/sonarwhal/sonarwhal)
- [x] Followed the [commit message guidelines](https://sonarwhal.com/docs/contributor-guide/getting-started/pull-requests/#commit-messagess)

For non-trivial changes, please make sure you also:

- [x] Added/Updated related documentation.
- [x] Added/Updated related tests.

## Short description of the change(s)
This commit update the documentation generated by `new-rule` for
packages mono rule and multi rules.

Fix #1013

### CODE REVIEW

File: packages/sonarwhal/src/lib/cli/wizards/new-rule.ts [hunk 1]
```diff
@@ -354,10 +358,22 @@ const generateRuleFiles = async (destination: string, data) => {
...
         const testPath = path.join(testFile.destination, `${rule.normalizedName}.ts`);
+        // e.g.: rule-typescript-config/docs/is-valid.ts
+        const docPath = path.join(docFile.destination, `${rule.normalizedName}.md`);
 
         await Promise.all([mkdirpAsync(path.dirname(rulePath)), mkdirpAsync(path.dirname(testPath))]);
 
         await Promise.all([writeFileAsync(rulePath, ruleContent), writeFileAsync(testPath, testContent)]);
+
+        if (data.isMulti) {
+            if (data.isMulti) {
```
Comment (molant) [inline packages/sonarwhal/src/lib/cli/wizards/new-rule.ts]: I think you have something wrong here with this double of checking the same thing.

File: packages/sonarwhal/src/lib/cli/wizards/templates/new-rule/readme.md.hbs [hunk 2]
```diff
@@ -1,24 +1,52 @@
...
 
 {{description}}
 
 ## Why is this important?
 
-Explain why this {{#if multi}}package{{else}}rule{{/if}} is important for your users
+Explain why this {{#if isMulti}}package{{else}}rule{{/if}} is important for your users
 
-## What does the {{#if multi}}package{{else}}rule{{/if}} check?
+## How to use {{#if isMulti}}these rules{{else}}this rule{{/if}}?
```
Comment (alrra) [inline packages/sonarwhal/src/lib/cli/wizards/templates/new-rule/readme.md.hbs]: I was thinking that maybe we should move this after `## Further Reading`?

Comment (molant) [reply packages/sonarwhal/src/lib/cli/wizards/templates/new-rule/readme.md.hbs]: I'm ok, but we will have to update all packages. I've opened https://github.com/sonarwhal/sonarwhal/issues/1018 to do this.

File: packages/sonarwhal/src/lib/cli/wizards/new-rule.ts (hunk 1 above)
Comment (sarvaje) [reply packages/sonarwhal/src/lib/cli/wizards/new-rule.ts]: Yes @molant, this is an error, it should be just one `if`

**Pull request merged**